### DISCUSSION TITLE
Search/Filter Request Statistic

### DISCUSSION THREAD

**Discussion by MoritzSalein** [ludeknovy/jtl-reporter#89] [Q&A] [answered]
Is it possible to filter the Request Statistic label with multiple values and/or values without the given string? E.g. all request without the value "Transaction" or all requests with "login" AND "logout".
![image](https://user-images.githubusercontent.com/83296020/134863794-19d86e24-623f-4e29-a1bc-bee455927a9d.png)

**Comment by ludeknovy** [comment, marked as answer]
Hi @MoritzSalein, currently not possible. But I like the idea and will try to add it.

**Reply by ludeknovy** [reply]
@MoritzSalein would that do it?

https://user-images.githubusercontent.com/13610612/134899706-c49241b0-93f7-4c39-a704-f04f47c3a8fd.mov

**Reply by MoritzSalein** [reply]
Yes, that's what I want.

**Reply by ludeknovy** [reply]
Btw do you expect here "AND" or "OR" operator? I guess "OR" makes more sense here, right?

**Reply by MoritzSalein** [reply]
I think an 'OR' makes more sense. I don't have an idea for an 'AND' at the moment. That was I mean in my first post was more an or as an and :-) .
